docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,356 @@
+
+To get started creating subfolders, you'll need to enable the `nestedFolders` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for managing dashboards and folders, to provide a more streamlined user experience.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10.0-nested-folders.png" max-width="750px" caption="Subfolders in Grafana" >}}
+
+## Dashboards and visualizations
+
+### The Canvas panel is GA
+
+<!-- Nathan Marrs -->
+
+We are promoting the canvas panel out of beta and into general availability. Over the past several months we have introduced several enhancements to the panel such as the ability to draw connections between elements, the ability to set the color and size of connections based on data, and the ability to add data links. We are excited to include canvas as a first class citizen in Grafana’s core panel library. To learn more about the Canvas panel, refer to [Canvas]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-canvas-service-graph.png" max-width="750px" caption="Canvas service graph" >}}
+
+### New Trend Panel
+
+<!-- Nathan Marrs -->
+
+_Available in Experimental in all editions of Grafana_
+
+We are excited to introduce a new panel that allows you to display trends where the x axis is not time. This new beta panel addresses gaps that were not solved by either the Time series or XY Chart panels.
+
+### Drag and Drop spreadsheets into Grafana
+
+_Available in Experimental in all editions of Grafana_
+
+<!-- Oscar Kilhed -->
+
+it's easier than ever to view local data in Grafana - welcome drag & drop!
+The drag & drop functionality allows you to upload your csv, excel or numbers files by simply dragging and dropping them into the query editor of the Grafana data source.
+
+Drag & drop feature is disabled by default. In order to use drag and drop, enable the `editPanelCSVDragAndDrop` feature toggle.
+
+As of Grafana 10, drag & drop supports the following scenarios:
+
+- Drag & drop files into the panel editor
+- Replace files in the panel editor
+- Default table panel creation
+
+Data is being stored in the dashboard json and has a 1MB size limit. To learn more about drag & drop, please refer to the official documentation.
+
+<!-- TODO: Add docs link above -->
+
+### Select your data source more easily
+
+<!-- Ivan Ortega & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
+
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
```

Comment:
@imatwawana since we will enable the toggle by default, it will be generally available. Just removing this sentence is enough for the What's new, isn't it? Thanks!

```suggestion
### Select your data source more easily

<!-- Ivan Ortega & Natalia Bernarte -->

Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
```